src/services/novel.js

```diff
@@ -0,0 +1,47 @@
+import models from '../database/models';
+
+const { User, Genre, Novel } = models;
+
+/**
+ *
+ *
+ * @param {object} novel
+ * @param {integer} authorId
+ * @returns {object} json
+ */
+const createNewNovel = async (novel, authorId) => {
+  const {
+    genre,
```

Comment:
hello, I think it would be better to have a predefined list of genres so that authors will not create random genres

Reply:
Alright! I'll make changes accordingly